chef/cookbooks/provisioner/recipes/dhcp_update.rb

```diff
@@ -21,9 +21,9 @@
              'if option arch = 00:06 {
        filename = "discovery/ia32/efi/bootia32.efi";
      } else if option arch = 00:07 {
-       filename = "discovery/x86_64/efi/bootx64.efi";
+       filename = "discovery/x86_64/efi/boot/grub2/x86_64-efi/core.efi";
      } else if option arch = 00:09 {
-       filename = "discovery/x86_64/efi/bootx64.efi";
+       filename = "discovery/x86_64/efi/boot/grub2/x86_64-efi/core.efi";
      } else if option arch = 00:0b {
        filename = "discovery/aarch64/efi/bootaa64.efi";
```

Comment:
I wonder why you're using grub2-mknetdir in the first place.. why not just use grub2-mkimage like originally suggest? is there a bugreport about grub2-mknetdir? looks like this isn't fixed upstream yet.